plugins/polio/js/src/pages/Budget/CreateBudgetStep/CreateBudgetStep.tsx

```diff
@@ -220,7 +220,7 @@ const CreateBudgetStep: FunctionComponent<Props> = ({
                 dataTestId="Test-modal"
                 onClose={() => null}
             >
-                {userHasTeam && (
```

Comment:
This also looks like it's part of another PR (maybe on my side)

Reply:
Solved